walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,112 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+import java.util.HashSet;
 
+/** Given a collection of events and a meeting request, returns a collection of Time Ranges for a
+ *      possible meeting handling cases for required and optional attendees.
+ */
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    // Returns no times if given a duration greater than a day.
```

Comment:
You want to write comments that explain the concept instead of stating the code. So this would be something like "No meetings possible for duration greater than a day."